server/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -34,14 +42,6 @@
 import org.elasticsearch.common.xcontent.XContentType;
 import org.elasticsearch.index.query.QueryShardContext;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Iterator;
-import java.util.List;
-import java.util.function.Function;
-
```

Comment:
oh boy, I guess this is because I configured checkstyle and the spotless within IntelliJ?

Reply:
I'm never sure about these! They don't make much difference to me and we have a long term plan to get rid of them in the code formatter. So I'm not too worried about them.